src/api/spec/features/webui/maintenance_workflow_spec.rb

```diff
@@ -0,0 +1,170 @@
+    fill_in('reason', with: 'really? ok')
+    click_button('accept_request_button')
+    # rubocop:disable Metrics/LineLength
+    expect(find(:css, '#action_display_0')).to have_text(/Submit update from package home:tom:\.\.\.o:Update \/ ProjectWi\.\.\._package to package MaintenanceProject:0 \/ ProjectWi\.\.\.o_Update/)
+    # rubocop:enable Metrics/LineLength
+    visit(project_show_path(project: 'MaintenanceProject:0'))
+    find(:link, 'Patchinfo present').click
+    find(:id, 'edit-patchinfo').click
+
+    page.evaluate_script('window.confirm = function() { return true; }')
+    find(:link, 'Update issues from sources').click
+    expect(page).to have_content('Patchinfo-Editor for MaintenanceProject:0')
+
+    expect(find(:id, 'summary')).to have_text('I want the update')
+
+    fill_in('summary', with: 'ProjectWithRepo_package is much better than the old one')
+    fill_in('description', with: 'Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing, Fixes nothing')
+    check('block')
+    fill_in('block_reason', with: 'locked!')
+    click_button('Save Patchinfo')
+
+    # summary and description are ok
+    expect(page).to have_no_css('span.ui-icon.ui-icon-alert')
+
+    expect(find(:css, 'span.ui-icon.ui-icon-info')).to have_text('Successfully edited patchinfo')
+    expect(find(:css, '.ui-state-error b')).to have_text('This update is currently blocked:')
+
+    click_link('MaintenanceProject')
+    click_link('open incident')
+
+    click_link('recommended')
+    click_link('edit-patchinfo')
+    uncheck('block')
+    expect(page).to have_css('input[id=block_reason][disabled]')
+    click_button 'Save Patchinfo'
+
+    logout
+
+    # add a additional fix to the incident
+    login(user)
+    visit project_show_path(project: 'home:tom:branches:ProjectWithRepo:Update')
+
+    click_link('Submit as update')
+
+    expect(find(:css, '.dialog h2')).to have_text('Submit as Update')
+    fill_in('description', with: 'I have a additional fix')
+    click_button('Ok')
+
+    logout
+
+    # let the maint-coordinator add the new submit to the running incident and cont
+    login(maintenance_coord_user)
+    visit project_show_path(project: 'MaintenanceProject')
+
+    click_link('open request')
+    # find("tbody tr:first-child a.request_link").click # It isn't necessary, it's only one
+    expect(find(:id, 'description-text')).to have_text('I have a additional fix')
+    click_link('Merge with existing incident')
+
+    # set to not existing incident
+    fill_in('incident_project', with: '2')
+    click_button('Ok')
+    expect(find(:css, 'span.ui-icon.ui-icon-alert')).to have_text('does not exist')
+
+    click_link('Merge with existing incident')
+    fill_in('incident_project', with: '0')
+    click_button('Ok')
+
+    expect(find(:css, 'span.ui-icon.ui-icon-info')).to have_text(/Set target of request.*to incident 0/)
+
+    click_button('accept_request_button')
+
+    # TODO: make it unique find(:link, "0").click
```

Comment:
That line was in the old test: [test/functional/webui/maintenance_workflow_test.rb](https://github.com/openSUSE/open-build-service/blob/af51d523be0696172682bb32d31f4da509d18a07/src/api/test/functional/webui/maintenance_workflow_test.rb#L129)